qualtran/bloqs/phase_estimation_of_quantum_walk.ipynb

```diff
@@ -76,8 +82,10 @@
     "    den = 1 + 2 ** m\n",
     "    norm = np.sqrt(2 / den)\n",
     "    return norm * np.sin(np.pi * (1 + np.arange(2**m)) / den)        \n",
-    "    \n",
-    "def phase_estimation(walk: QubitizationWalkOperator, m: int) -> cirq.OP_TREE:\n",
+    "\n",
+    "\n",
+    "@attrs.frozen\n",
+    "class ResourceStatePhaseEstimation(GateWithRegisters):\n",
```

Comment:
can we move this code into the library

Reply:
Yeah, I'm working on it and I've discovered there are typos in the paper. I've filed a new issue to track that. I can create a PR where we haven't tested the simulation yet and work on fixing the typos later if that helps.